### PR TITLE
Auto take judgment user to file checks results

### PR DESCRIPTION
**Pull request by techncl**

### CODE REVIEW

File: app/controllers/FileChecksController.scala [hunk 1]
```diff
@@ -36,16 +36,26 @@ class FileChecksController @Inject()(val controllerComponents: SecurityComponent
...
+            consignmentId, frontEndInfoConfiguration.frontEndInfo, request.token.name, isJudgmentUser = false
+          )).uncache()
+        } else {
+          Ok(views.html.standard.fileChecksProgress(consignmentId, frontEndInfoConfiguration.frontEndInfo, request.token.name)).uncache()
+        }
       }
   }
 
-  def judgmentProcessingPage(consignmentId: UUID): Action[AnyContent] = judgmentTypeAction(consignmentId) { implicit request: Request[AnyContent] =>
+  def judgmentsRecordProcessingPage(consignmentId: UUID): Action[AnyContent] = judgmentTypeAction(consignmentId) { implicit request: Request[AnyContent] =>
```
Comment (TomJKing) [inline app/controllers/FileChecksController.scala:49]: Would rename slightly for consistency: `judgmentRecordProcessingPage`

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment. Will the e2e tests need updating for this change?

**Comment by techncl** [comment]
> Looks good. One minor comment. Will the e2e tests need updating for this change?

Yes, I've made a [PR for it](https://github.com/nationalarchives/tdr-e2e-tests/pull/169)

**Review by TomJKing — DISMISSED** [review]
Looks good. Thanks

**Review by TomJKing — APPROVED** [review]

**Pull request merged**